src/lib/connect.tsx

```diff
@@ -58,37 +54,33 @@ module.exports = function connect(generateProps) {
         return {
           subscription: this.subscription
         };
-      },
+      }
 
       /**
        * Handle render from parent (not called during mounting) during default render flow
        * @param {Object} nextProps
        * @param {Object} nextContext
        */
-      componentWillReceiveProps(nextProps, nextContext) {
+      componentWillReceiveProps(nextProps: any, nextContext: any) {
         this.setState({
           props: generateProps(nextContext, nextProps)
         });
-      },
+      }
 
       /**
        * Determine if component should render based on result of 'generateProps'.
        * If not rendering, manually notifies all connected child containers instead of relying on default render flow
-       * @param {Object} nextProps
-       * @param {Object} nextState
-       * @param {Object} nextContext
-       * @returns {Boolean}
        */
-      shouldComponentUpdate(nextProps, nextState, nextContext) {
+      shouldComponentUpdate(_nextProps: any, nextState: any): boolean {
```

Comment:
Is the underscore for unused parameter a typescript convention?

Reply:
Yes:
https://github.com/Microsoft/TypeScript/pull/9464